r/adbcsnowflake/src/Makevars.in

```diff
@@ -15,8 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
-PKG_CPPFLAGS=-I$(CURDIR)/src -DADBC_EXPORT=""
-PKG_LIBS=-L$(CURDIR)/go -ladbc_driver_snowflake -lresolv @libs@
```

Comment:
didn't we find that we needed `-lresolv` for macOS systems?

Reply:
Configure[ adds it](https://github.com/apache/arrow-adbc/pull/1220/files#diff-f38df91821ffbca4a02ed033210d891a0e2dc9e8b3e30e89becad3965cc15365R134) depending on os.

Reply:
ugh pending review comments... ^^